Workbooks/UpdateCompliance/AlertsTargetBuild/Alerts for Target build.workbook

```diff
@@ -0,0 +1,571 @@
+{
+  "version": "Notebook/1.0",
+  "items": [
+    {
+      "type": 9,
+      "content": {
+        "version": "KqlParameterItem/1.0",
+        "parameters": [
+          {
+            "id": "95c423b6-142d-4aa4-bbe6-0d408498a3d7",
+            "version": "KqlParameterItem/1.0",
+            "name": "_ShowPills",
+            "type": 1,
+            "description": "Hide pills",
+            "isRequired": true,
+            "isHiddenWhenLocked": true,
+            "criteriaData": [
+              {
+                "criteriaContext": {
+                  "operator": "Default",
+                  "resultValType": "static",
+                  "resultVal": "no"
+                }
+              }
+            ],
+            "timeContext": {
+              "durationMs": 86400000
+            }
+          }
+        ],
+        "style": "pills",
+        "queryType": 0,
+        "resourceType": "microsoft.operationalinsights/workspaces"
+      },
+      "name": "parameters - 3"
+    },
+    {
+      "type": 9,
+      "content": {
+        "version": "KqlParameterItem/1.0",
+        "parameters": [
+          {
+            "id": "03761a72-d969-426b-ba54-d1e709610b2d",
+            "version": "KqlParameterItem/1.0",
+            "name": "_WorkbookContext",
+            "label": "Workbook Context",
+            "type": 1,
+            "description": "Whether workbook is opened standalone or as part of the Hub.",
+            "isRequired": true,
+            "query": "print(\"Focused\")",
```

Comment:
could use criteria data with default condition or text with hard coded value of 'Focused' instead of making a network call to logs